### DISCUSSION TITLE
Docker Image Build Failed on Google Cloud Run next:13.4.16

### DISCUSSION THREAD

**Discussion by AnilUchadiya** [vercel/next.js#54155] [Help]
### Summary

Error: ` Revision 'image-xxx' is not ready and cannot serve traffic. The user-provided container failed to start and listen on the port defined provided by the PORT=8080 environment variable.`

When I was trying to deploy the build of our NextJs application on google cloud run, it failed to start the container, It was working fine on the local machine where we were using older NextJS:13.4.6 version .
After reverting the version in the package.json file, it worked.

**Comment by Aakash180192** [comment]
Thanks!!
I was also getting the same issue, reverting to previous version solved the issue.

**Comment by majksner** [comment]
Related to:
https://github.com/vercel/next.js/issues/54133
https://github.com/vercel/next.js/issues/53367
